engines/dradis-api/config/routes.rb

```diff
@@ -1,8 +1,17 @@
 Dradis::CE::API::Engine::routes.draw do
+  concern :with_comments do
+    resources :comments, only: [:index, :create]
+  end
+
   scope path: :api do
     defaults format: 'json' do
       scope module: :v1, constraints: Dradis::CE::API::RoutingConstraints.new(version: 1, default: true) do
-        resources :issues
+        resources :issues, concerns: :with_comments
+        resources :evidence, only: [], concerns: :with_comments
+        resources :notes, only: [], concerns: :with_comments
```

Comment:
what about this `only: []` ?

Reply:
We don't want to add a shallow routes for evidence and notes. ex: `/api/notes/1/`. Although I actually prefer this shallow path than the one with `nodes`, but this isn't the PR for that.